x-pack/plugins/apm/server/routes/historical_data.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setupRequest } from '../lib/helpers/setup_request';
+import { hasHistoricalAgentData } from '../lib/services/get_services/has_historical_agent_data';
```

Comment:
Should this file still be located in `get_services`? Perhaps move it to a "shared/" folder or something like that

Reply:
I moved to the routes level inside `/historical_data` folder, so it lives at the same level as the API